op-deployer/pkg/deployer/opcm/read_superchain_deployment.go

```diff
@@ -6,21 +6,23 @@ import (
 )
 
 type ReadSuperchainDeploymentInput struct {
-	OPCMAddress common.Address `abi:"opcmAddress"`
+	OPCMAddress           common.Address `abi:"opcmAddress"` // TODO: Remove OPCMAddress field when OPCMv1 gets deprecated
```

Comment:
General note that `// TODO` comments should take the form of `// TODO(#XXXXX)` where `XXXXX` is an issue number so the TODO can be tracked